cli/generate_models.py

```diff
@@ -210,38 +211,39 @@ def __init__(self, value: str | dict) -> None:
             **COMMON_FIELD_CLASSES,
             **RELATION_FIELD_CLASSES,
         }
-        self.contraints = {}
+        self.constraints = {}
         self.in_array_constraints = {}
         if isinstance(value, str):
             self.type = value
         else:
-            self.type = value.get("type", "")
+            self.type = value.pop("type")
             if self.type in RELATION_FIELD_CLASSES.keys():
-                self.to = To(value.get("to", {}))
-                self.on_delete = value.get("on_delete")
+                self.to = To(value.pop("to"))
+                self.on_delete = value.pop("on_delete", None)
+                if self.type in ("relation-list", "generic-relation-list") or value.get(
+                    "sql"
+                ):
+                    self.is_view_field = True
```

Comment:
@r-peschke this is the important section where it is determined whether a field is a view or table field. Please check if this conforms with the current/planned models schema or if there are other cases we need to account for

Reply:
No, this is far away from current or planned case decision. I'll will present a table soon